cfme/tests/configure/test_default_filters.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+import cfme.configure.settings as st
+from cfme.fixtures import pytest_selenium as sel
+from utils.providers import setup_a_provider
+
+
+@pytest.fixture(scope="module")
+def setup_first_provider():
+    setup_a_provider(validate=True, check_existing=True)
+
+
+@pytest.mark.parametrize('filters',
+                        [[['Cloud', 'Instances', 'Images', 'Platform / Amazon']]])
+def test_cloudimage_defaultfilters(filters, setup_first_provider):
+    df = st.Defaultfilter(name='Platform / Amazon')
+    df.update({'filters': [(k, True) for k in filters]})
+    sel.force_navigate('clouds_images_filter_folder', context={'folder_name': 'Global Filters'})
+    assert sel.is_displayed_text(df.name), "Default Filter settings Failed!"
+
+
+@pytest.mark.parametrize('filters',
+                        [([['Cloud', 'Instances', 'Instances', 'Platform / Openstack']])])
```

Comment:
Maybe some explanation why there is so much of nesting / parametrization format could come handy.